### DISCUSSION TITLE
Code editor property UI

### DISCUSSION THREAD

**Discussion by bjarnef** [umbraco/Umbraco-CMS#16673] [Features and ideas]
**Is your feature request related to a problem? Please describe.**
The current backoffice doesn't implements code editor, which is often useful to insert/embed third party scripts, iframe etc. and offer syntax highlighting.
At the moment we would have to use Contentment or implement a custom property editor similar to implementation in this PR:
https://github.com/umbraco/Umbraco-CMS/pull/15314

**Describe the solution you'd like**
A simple code editor using Monaco editor and eventually a few settings to configure.

**Describe alternatives you've considered**

**Additional context**

**Comment by enkelmedia** [comment]
We sometimes provide a "Code" block for editors where I historically built my own property editor. This would be a great addition. It would probably also be super-easy to build the property editor as a package in the future and just reuse the `umb-code-editor`-element.
